modules/phase_field/examples/ebsd_reconstruction/IN100-111grn.i

```diff
@@ -123,17 +135,11 @@
   [../]
   [./grain_tracker]
     type = GrainTracker
-    threshold = 0.1
-    convex_hull_buffer = 0.0
-    use_single_map = false
-    enable_var_coloring = true
-    condense_map_info = true
-    connecting_threshold = 0.05
-    execute_on = 'initial timestep_end'
-    flood_entity_type = ELEMENTAL
-    halo_level = 2
-    bubble_volume_file = IN100-grn-vols.txt
+    threshold = 0.2
+    connecting_threshold = 0.2
+    execute_on = 'initial timestep_begin'
```

Comment:
This is awesome the way this cleans up! No need to set the `connecting_threshold` parameter if it's the same as `threshold`.